runtime/lua/vim/lsp/handlers.lua

```diff
@@ -237,6 +237,11 @@ RCS[ms.textDocument_inlayHint] = function(...)
   return vim.lsp.inlay_hint.on_inlayhint(...)
 end
 
+--- @private
+RCS[ms.textDocument_linkedEditingRange] = function(...)
```

Comment:
Instead of adding the handler here, we can just pass the `handler` in `client:request`.

Reply:
https://github.com/neovim/neovim/pull/33440/commits/269688773269356c3af306fa695782411f2d73dc might provide some insight on how to do it.